flexget/plugins/clients/transmission.py

```diff
@@ -756,7 +756,18 @@ def _make_torrent_options_dict(self, config, entry):
             change['queuePosition'] = opt_dic['queue_position']
 
         if 'labels' in opt_dic:
-            change['labels'] = opt_dic['labels']
+            labels = set()
+            for obj in [config, entry]:
+                obj_labels = obj.get('labels', [])
+                if not isinstance(obj_labels, list):
+                    obj_labels = [obj_labels]
+                labels.update(obj_labels)
+            labels = [entry.render(label) for label in labels]
+            labels = [re.sub(r"(\s),|,(\s)", r"\1\2", label).replace(",", " ") for label in labels]
```

Comment:
What is the regex meant to do? It strips out commas that have whitespace next to them? Why do that separate than the regular comma replacement?

Reply:
If a comma is next to a whitespace, it's just removed, otherwise it's replaced with space.

It's to reduce unnecessary whitespace. So `1,2, 3, 4 , 5` becomes `1 2 3 4  5` instead of `1 2  3  4   5`.

Reply:
I know it's sorta nitpicky and doesn't matter much, but my inclination would be to make the code simple and leave the formatting to the user. I think replacing the ',' is fine since things could break without that, but really it should be up to the user to format the label how they want. If you really want to avoid duplicate whitespace while removing commas it could just be `re.sub("\s*,\s*", " ", label)` which would replace commas and all surrounding whitespace with just a space, and then the second `replace` call isn't needed.

I think also a comment is warranted here like `# Transmission doesn't allow commas in labels`, just so we remember why that's happening in the future.

Reply:
My intent was to not change user formatting too much.
`re.sub("\s*,\s*", " ", label)` would remove user added whitespace. Also `\s` matches more than just space.
`label.replace(",", " ")` would create double spaces.
`label.replace(",", "")` would merge words.

And user doesn't control formatting in text added with templating.
Titles with commas usually have just comma followed by space, where simple replacement would work.

Reply:
Maybe `re.sub(", ?", " ", label)` then? I'm good with whatever you think is best here though.

Reply:
Ok, I used this. It should be good enough for most use cases.